apps/server/src/workers/taskPlanner.ts

```diff
@@ -115,37 +111,18 @@ export class TaskPlanner {
         .delete()
         .from(Chunk)
         .where('taskId IN (:...ids)', { ids: floatTasks.map((t) => t.id) })
-        // Where start is after task.start
-        .andWhere('start >= :start', { start: task.chunkInfo.start })
+        // Where chunks are not done
+        .andWhere('isDone = :isDone', { isDone: false })
```

Comment:
We may also want to only delete chunks from the future.

Reply:
Good point! But I need to test it better to make sure that nothing crashes with such an assumption.
Let's have it like that for now, I'll keep it in mind 💯

Reply:
You can first write tests with assertions (for them to be real tests) and then change this part and check if everything is smooth still.